src/gdb/gdb.c

```diff
@@ -469,7 +469,7 @@ static void read_sr(int sr)
 		      : "=r"(val)
 		      : "r"(sr)
 		      : "a3", "memory");
-	sregs[sr] = val;
+	//sregs[sr] = val;
```

Comment:
intentional or can it be removed ?

Reply:
Can be removed and actually  was in "cleanup" commit.